hw/ip/uart/doc/_index.md

```diff
@@ -201,10 +201,9 @@ UART module has a few interrupts including general data flow interrupts
 and unexpected event interrupts.
 
 #### tx_watermark / rx_watermark
-If the TX or RX FIFO level becomes greater than or equal to their respective
-high-water mark levels (configurable via {{< regref "FIFO_CTRL.RXILVL" >}} and
-{{< regref "FIFO_CTRL.TXILVL" >}}), interrupts `tx_watermark` or `rx_watermark` are raised to
-inform SW.
+If the TX FIFO level becomes smaller than the TX water mark level (configurable via {{< regref "FIFO_CTRL.RXILVL" >}} and {{< regref "FIFO_CTRL.TXILVL" >}}), the `tx_watermark` interrupt is raised to inform SW.
+If the RX FIFO level becomes greater than or equal to RX water mark level (configurable via {{< regref "FIFO_CTRL.RXILVL" >}} and {{< regref "FIFO_CTRL.TXILVL" >}}), the rx_watermark` interrupt is raised to inform SW.
+
 
 #### tx_overflow / rx_overflow
```

Comment:
please also update tx_overflow to TxEmpty

Reply:
ah good catch, thanks.

Reply:
updated.